src/testFixtures/java/org/opensearch/neuralsearch/BaseNeuralSearchIT.java

```diff
@@ -1504,23 +1505,32 @@ protected void wipeOfTestResources(
         final String modelId,
         final String searchPipeline
     ) throws IOException {
-        if (ingestPipeline != null) {
-            deletePipeline(ingestPipeline);
-        }
-        if (searchPipeline != null) {
-            deleteSearchPipeline(searchPipeline);
-        }
-        if (modelId != null) {
-            try {
-                deleteModel(modelId);
-            } catch (AssertionError e) {
-                // sometimes we have flaky test that the model state doesn't change after call undeploy api
-                // for this case we can call undeploy api one more time
-                deleteModel(modelId);
+        try {
+            if (ingestPipeline != null) {
```

Comment:
I agree with the discussion below to move cleanup to an @After block, but I want to point out that the root cause is that our integ tests call this method and pass in static constant Strings directly into ingestPipeline/searchPipeline/indexName regardless of whether or not the resources were created like here: https://github.com/opensearch-project/neural-search/blob/main/src/test/java/org/opensearch/neuralsearch/processor/TextEmbeddingProcessorIT.java#L76

In those conditions these blocks are never triggered. Not saying we need to change that right now, but pointing out with an @After block we still don't address the root cause here, just move the exception where we can't see it?